pyproject.toml

```diff
@@ -115,6 +115,14 @@ sphinx = "6.1.3"
 tensorflow = {version = "2.12.0", markers="sys_platform != 'darwin'"}
 tensorflow-macos = {version = "2.12.0", markers="sys_platform == 'darwin'"}
 tensorflow-metal = {version = "0.8.0", markers="sys_platform == 'darwin' and platform_machine == 'arm'"}
+tensorflow-io-gcs-filesystem = [
```

Comment:
We need to force to 0.34.0, since it has a larger support of python versions and os/architectures.